engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -250,11 +252,20 @@ private void initRenderGraph() {
         renderGraph.addNode(blurPassesNode, "blurPassesNode");
         renderGraph.addNode(finalPostProcessingNode, "finalPostProcessingNode");
 
+        initDefaultStateChanges();
+
         List<Node> orderedNodes = renderGraph.getNodesInTopologicalOrder();
         RenderTaskListGenerator generator = new RenderTaskListGenerator();
         renderPipelineTaskList = generator.generateFrom(orderedNodes);
     }
 
+    private void initDefaultStateChanges() {
+        SetViewportSizeOf.setDefaultInstance(new SetViewportSizeOf(frameBuffersManager));
+        BindFBO.setDefaultInstance(new BindFBO());
+        SetWireframe.setDefaultInstance(new SetWireframe());
```

Comment:
I am sorry but this way of doing things didn't quite "click" with me. 

The SetWireframe state change is quite simple due to its boolean nature and shouldn't need any pre-initialization: initialization on instantiation is enough.

For the other two state changes, here in WorldRendererImpl.initDefaultStateChanges() I'd just write:
```
SetViewportSizeOf.setFrameBuffersManager(fbm);
BindFBO.setFrameBuffersManager(fbm);
```

In both StateChanges there should be:

`private static FrameBuffersManager fbm;`

and

public setFrameBuffersManager(FrameBuffersManager fbm) {
    this.fbm = fbm;
}

In so doing, the constructors won't need the FBM parameter in them:

```
SetViewportSizeOf(String fboName);
BindFBO(String fboName);
```

and we won't need the setDefaultInstance() method.